app/frontend/src/features/communities/leaveReference/formSteps/Appropriate.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Card,
+  CardContent,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Typography,
+  useMediaQuery,
+  useTheme,
+} from "@material-ui/core";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import Divider from "components/Divider";
+import TextBody from "components/TextBody";
+import {
+  APPROPRIATE_BEHAVIOR,
+  APPROPRIATE_EXPLANATION,
+  APPROPRIATE_QUESTION,
+  NEXT,
+  PRIVATE_ANSWER,
+  REFERENCE_FORM_HEADING,
+  SAFETY_PRIORITY,
+} from "features/communities/constants";
+import {
+  ReferenceContextFormData,
+  useReferenceData,
+} from "features/communities/leaveReference/ReferenceDataContext";
+import {
+  ReferenceFormProps,
+  useReferenceStyles,
+} from "features/communities/leaveReference/ReferenceForm";
+import React from "react";
+import { Controller, useForm } from "react-hook-form";
+import { useHistory, useParams } from "react-router-dom";
+import { leaveReferenceBaseRoute } from "routes";
+
+export default function Appropriate({ user }: ReferenceFormProps) {
+  const history = useHistory();
+  const theme = useTheme();
+  const classes = useReferenceStyles();
+  const isMdOrWider = useMediaQuery(theme.breakpoints.up("md"));
+  const { referenceType, hostRequest } = useParams<{
+    referenceType: string;
+    hostRequest?: string;
+  }>();
+  const { data, setValues } = useReferenceData()!;
+  const { control, handleSubmit, errors } = useForm<ReferenceContextFormData>({
+    defaultValues: {
+      wasAppropriate: data.wasAppropriate,
+    },
+  });
+
+  const onSubmit = (values: ReferenceContextFormData) => {
+    setValues(values);
+    hostRequest
+      ? history.push(
+          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/${hostRequest}/rating`
+        )
+      : history.push(
+          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/rating`
+        );
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)} className={classes.form}>
+      <Typography variant="h1">
```

Comment:
```suggestion
      <Typography variant="h2">
```
Same comment as in `Rating`. If you need h1 size, it's better to style it that way and keep the semantic correct for accessibility (otherwise we skipped a heading level which is an anti-pattern)